app/stac_api/models.py

```diff
@@ -197,27 +190,469 @@ def update_geoadmin_variants(self, asset_geoadmin_variant, asset_proj_epsg, asse
                 "Error when updating collection's summaries values due to asset update."
             ))
 
-    def update_temporal_extent(self, item_properties_start_datetime, item_properties_end_datetime):
+    def insert_item(
+        self, old_start_datetime, new_start_datetime, old_end_datetime, new_end_datetime, item_id
+    ):
         '''
-        updates the collection's temporal extent when item's are update.
-        :param item_properties_datetime: item's value for properties_datetime.
-        This function checks, if the corresponding parameter of the collection
-        needs to be updated. If so, it will be either updated or an error will
-        be raised, if updating fails.
+        This function is called from within update_temporal_extent() when a new
+        item is inserted to the collection.
+        :param old_start_datetime: item's old value for properties_start_datetime
+        (in this case None, as it is a new item)
+        :param new_start_datetime: item's updated value for properties_start_datetime
+        :param old_end_datetime: item's old value for properties_end_datetime
+        (in this case None, as it is a new item)
+        :param new_end_datetime: item's updated value for properties_end_datetime
+        :param item_id: the id of the item being treated
         '''
-
+        logger.debug('Inserting item %s in collection %s', item_id, self.collection_name)
```

Comment:
Having the datetimes value in log might be useful in future

Reply:
added datetime values (where applicable) and extra parameter with collection_name in all loggers